weather_dl_v2/license_deployment/fetch.py

```diff
@@ -90,20 +127,22 @@ def fetch_request_from_db():
 
 def main():
     logger.info("Started looking at the request.")
+    error_map = ThreadSafeDict()
     with ThreadPoolExecutor(concurrency_limit) as executor:
         while True:
             # Fetch a request from the database
             request = fetch_request_from_db()
 
             if request is not None:
-                executor.submit(make_fetch_request, request)
+                executor.submit(make_fetch_request, request, error_map)
             else:
                 logger.info("No request available. Waiting...")
                 time.sleep(5)
 
-            # Check if the maximum concurrency level has been reached
+            # # Check if workers are busy
```

Comment:
remove extra `#`.

Reply:
nit: can we have more descriptive comment here.

Reply:
fixed